pontoon/sync/tests/test_sync_projects.py

```diff
@@ -92,7 +92,10 @@ def test_invalid_slugs(self):
             force=False
         )
 
-        assert_equal(self.command.stderr.getvalue(), 'Couldn\'t find projects with following slugs: aaa, bbb')
+        assert_equal(
+            self.command.stderr.getvalue(),
+            'Couldn\'t find projects with following slugs: aaa, bbb'
```

Comment:
Nit, unrelated to PEP8, and personal opinion: I'd prefer `"Couldn't find projects with following slugs"`.